Life-Pill/pharmacy-manager-web-app
# Update Branch Management Layout and Visuals

# Pull Request: Update Branch Management Layout and Visuals

## Changes Made
- Fixed the sidebar active item bug in the SummaryCards component.
- Added the BranchManagement component to Branches.tsx and updated the PharmacyBranch interface.
- Updated the layout of BranchManagement in the branch-management-dashboard.
- Created and updated branch cards for better visuals.
- Refactored the branch management layout for improved code structure and readability.

## Manual Testing Done
- Checked the sidebar to ensure the active item bug is fixed.
- Verified that the BranchManagement component is added and functioning correctly in Branches.tsx.
- Tested the layout updates in the branch-management-dashboard to confirm visual improvements.
- Examined the branch cards for better visuals and ensured they display the necessary information correctly.
- Reviewed the refactored branch management layout for improved code organization and readability.

## Screenshots of Changes Made
![image](https://github.com/Life-Pill/pharmacy-manager-web-app/assets/119271523/77f76683-0fe3-4d16-bd67-250081f5865f)

## Contributor
- Tharindu Jayawardhana
- Frontend Developer at LifePill
